Config.in

```diff
@@ -194,6 +194,8 @@ menu "Emulators"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/emulators/cemu/cemu/Config.in"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/emulators/cemu/cemu-hook/Config.in"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/emulators/cemu/cemutil/Config.in"
+  source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/emulators/xash3d/xash3d-fwgs/Config.in"
```

Comment:
Xash3D is not an emulator.

Reply:
There were no other sections when I wrote this initially but this got fixed by https://github.com/batocera-linux/batocera.linux/pull/2702

Reply:
Nice. Thanks